gobblin-runtime/src/main/java/gobblin/runtime/util/MetricNames.java

```diff
@@ -20,7 +20,7 @@
  */
 public class MetricNames {
   public static class LauncherTimings {
-    public static String BASE_NAME = "gobblin.launcher.timings";
+    public final static String BASE_NAME = "gobblin.launcher.timings";
     public final static String WORK_UNITS_CREATE = MetricRegistry.name(BASE_NAME, "work.units.create");
```

Comment:
I changed all of them to `static final` to be consistent with everywhere elsein #146.